ui/src/app/components/common-env-conf-panels/environment-informed-action-panel.tsx

```diff
@@ -96,20 +88,56 @@ export const EnvironmentInformedActionPanel = ({
   onPause,
   onResume,
   environmentChanged = false,
-}: PanelProps) => (
-  <FlexRow style={styles.environmentInformedActionPanelWrapper}>
-    {appType === UIAppType.JUPYTER && (
-      <StartStopEnvironmentButton {...{ status, appType, onPause, onResume }} />
-    )}
-    <CostInfo
-      {...{
-        analysisConfig,
-        creatorFreeCreditsRemaining,
-        environmentChanged,
-        workspace,
-      }}
-      currentUser={profile.username}
-      isGKEApp={appType !== UIAppType.JUPYTER}
-    />
-  </FlexRow>
-);
+}: PanelProps) => {
+  const costEstimatorStyle: CSSProperties = {
+    padding: '.495rem .75rem',
+    justifyContent: 'space-evenly',
+    flexGrow: 1,
+    ...(environmentChanged
+      ? {
+          backgroundColor: colorWithWhiteness(colors.warning, 0.9),
+        }
+      : {}),
+  };
+
+  // with a PD, the "drawn from" text is too crowded
+  const showCostsDrawnFromWithCosts = !analysisConfig.diskConfig.detachable;
+
+  return (
+    <FlexColumn>
+      <FlexRow style={styles.environmentInformedActionPanelWrapper}>
+        {appType === UIAppType.JUPYTER && (
+          <StartStopEnvironmentButton
+            {...{ status, appType, onPause, onResume }}
+          />
+        )}
+        <EnvironmentCostEstimator
+          {...{ analysisConfig }}
+          data-test-id='cost-estimator'
+          isGKEApp={appType !== UIAppType.JUPYTER}
+          style={costEstimatorStyle}
+        />
+        {showCostsDrawnFromWithCosts && (
```

Comment:
Overall, this looks great, but my dataproc cluster seems a bit off:
<img width="722" alt="image" src="https://github.com/all-of-us/workbench/assets/24514630/81bb2b7c-436a-45aa-9125-01c754f8c573">

https://github.com/all-of-us/workbench/assets/24514630/25840efd-db12-4315-affe-9ac4df5a4b7b




6b94980">

Reply:
https://github.com/all-of-us/workbench/assets/24514630/74ce6b63-e3bb-45b3-b2b8-c4c3183f29da